examples/AzureSDKSmokeTest/Podfile

```diff
@@ -26,7 +26,10 @@
 # --------------------------------------------------------------------------
 
 platform :ios, '12.0'
-use_frameworks!
+
+if ENV['SWIFT_USE_FRAMEWORKS'] != nil
+  use_frameworks!
+end
 
```

Comment:
@bsiegel this will allow us to run `pod install` with an environment variable set to test static and dynamic modes. I have tested both and they work. The other option would be to have two Podfiles which you'd have to swap names to use (since you can't configure the name of the podfile) or simply two separate projects. This seemed the simplest path. Let me know what you think.

Reply:
Yep, that appears to work just fine. 👍